BuildScript.Nuke/SemanticVersion.cs

```diff
@@ -72,9 +72,12 @@ public class SemanticVersion
   /// </remarks>
   public string VersionWithoutPreRelease { get; }
 
-  public SemanticVersion (string version)
+  public SemanticVersion (string version, bool isLocalBuild)
   {
     Version = version;
+    if (isLocalBuild)
+      Version = $"{Version.Split('-')[0]}-x.9.{DateTime.Now:yyMMdd-HHmmss}";
```

Comment:
`versionMatch` contains all parts needed to build the local version, please don't split here.